sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveDDLSuite.scala

```diff
@@ -205,7 +205,7 @@ class HiveCatalogedDDLSuite extends DDLSuite with TestHiveSingleton with BeforeA
       spark.sql("CREATE VIEW v AS SELECT STRUCT('a' AS `a`, 1 AS b) q")
       checkAnswer(spark.table("v"), Row(Row("a", 1)) :: Nil)
 
-      spark.sql("ALTER VIEW v AS SELECT STRUCT('a' AS `b`, 1 AS b) q1")
+      spark.sql("ALTER VIEW v AS SELECT STRUCT('a' AS `c`, 1 AS b) q1")
```

Comment:
Ah, I see. Since we don't allow duplicate names in a top-level, it seems we need to follow it in this case, too. cc: @cloud-fan 
```
scala> spark.sql("CREATE VIEW v1 AS SELECT 'a' AS `a`, 1 AS b")
scala> spark.sql("ALTER VIEW v1 AS SELECT 'a' AS `b`, 1 AS b")
org.apache.spark.sql.AnalysisException: Found duplicate column(s) in the view definition: `b`
```

Reply:
So you want me to add this test ?

Reply:
This is out-of-scope in this PR. Do you wanna work on it? If so, please feel free to file jira for it.